src/bot.js

```diff
@@ -16,14 +16,16 @@ const retweet = () => {
     q: config.query,
     result_type: config.result_type,
     lang: config.lang,
-    tweet_mode: "extended"
+    tweet_mode: 'extended'
   };
 
   TwitterBot.get('search/tweets', params, (err, data) => {
     // when no errors
     if (!err) {
-      if (data.statuses[0].full_text.split('#').length - 1 === 1) {
+            const full_text_data = data.statuses[0].full_text ? data.statuses[0].full_text : undefined;
+            if ((full_text_data <140 ) && (full_text_data.split('#').length - 1 === 1) && (full_text_data.toLowerCase().includes("#100daysofcode"))){
```

Comment:
@ShaunSHamilton here is the optional chaining implementation.

Reply:
This does not do the same as my above suggestion.

I suggest you run this code through some example datasets like:

```javascript
const data = {};
const data = { statuses: [] };
const data = { statuses: [ {} ] };
const data = { statuses: [ { full_text: '' } ] };
```

JavaScript has many sneaky runtime errors that can crash programs/scripts. _We should probably get Naomi to convert this to TS_ 🙃

Reply:
I could do this, once this PR is merged.